cryptotracker/static/CCGraph.js

```diff
@@ -7,12 +6,13 @@ var T = 1000000000000;
 var priceList = [];
 var dates = [];
 
+
 class ChartBuilder {
-  constructor(interval) {
+  constructor(INTERVAL) {

```

Comment:
Typically global constants are only in all caps. `this.interval` is a class property so it should remain CamelCase since its not the actual global itself.

Reply:
Oh my mistake, I was changing things around to get it to work/troubleshoot and then did a replace all on interval.